src/machines/java/com/enderio/machines/common/init/MachineBlocks.java

```diff
@@ -142,7 +144,7 @@ public class MachineBlocks {
         .setLootTable((l,t) -> MachinesLootTable.copyStandardComponentsWith(l, t, EIODataComponents.STORED_ENTITY.get()))
         .setBlockStateProvider(MachineModelUtil::progressMachineBlock)
         .addBlockTags(BlockTags.NEEDS_IRON_TOOL, BlockTags.MINEABLE_WITH_PICKAXE)
-        .createBlockItem(ITEM_REGISTRY,
+        .createBlockItem(ITEM_REGISTRY, b -> new BlockItem(b, new Item.Properties().component(EIODataComponents.STORED_ENTITY, StoredEntityData.EMPTY)),
```

Comment:
What is the purpose of this change?

Reply:
This is so our tooltip logic shows that this item needs a soul to work in jei. It doesn't show up anywhere else, just jei and creative inv. I would keep it like this since a machine with an empty component can be seen as one that needs a soul, while a machine without the component is one that will work without, but can potentially use one.